### ISSUE TITLE
How to auto launch file viewer after file transfer is complete when In-Call

### ISSUE THREAD

**Issue by GoogleCodeExporter** [BinaryBlue/imsdroid#168]
```
What steps will reproduce the problem?
1. We are "in-call" and try to share content e.g. a .jpg file, 
2. After file transfer is complete, how to auto launch another activity say a 
JPG viewer to view the file that was just transferred?
3. I tried to launch my activity from MyMsrpSession.java takeIncomingSession 
(case FileTransfer) but does not work. My code is shown below:
Intent i = new Intent();
i.setAction(Intent.ACTION_VIEW);
i.setClassName("android.JPGViewer", "android.JPGViewer.JPGViewer");
Activity a = new Activity();
a.startActivity(i);

What is the expected output? What do you see instead?
I expect that once the file transfer is complete the new jpg viewer activity 
should be auto started. However, I get the following error message.
Error: Can't create handler inside thread that has not called Looper.prepare()

What version of the product are you using? On what operating system?
IMSDroid/v1.2.353 (doubango r539) on Android 2.2

Please provide any additional information below.


```

Original issue reported on code.google.com by `kow...@gmail.com` on 7 Feb 2011 at 9:12

**Comment by GoogleCodeExporter** [comment]
```
Also is there a way to know when file transfer is complete in ScreenAV.java. 
Please provide any suggestions that you think might work better.
```

Original comment by `kow...@gmail.com` on 7 Feb 2011 at 9:15

**Comment by GoogleCodeExporter** [comment]
```
One other issue I observed is that after the file transfer is complete the 
"In-Call" activity is moved to the status bar at the top i.e. I need to drag it 
down and click on the IMSDroid option to again have it come to the foreground. 
How can I avoid this? i.e. after the file transfer the "in-call" activity needs 
to be displayed on screen.
```

Original comment by `kow...@gmail.com` on 7 Feb 2011 at 9:52

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 10 Feb 2011 at 12:04
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
```
Hi Everybody.
I want to know how to get the reiceved message(incoming message)
and in OnInviteEvent method we must call sendMessage method and in the same 
onInviteEvent method we must call a takeIncomingSession method.
and in which cases they  called
Pleease write smth here.


```

Original comment by `timeandr...@time.com` on 20 Feb 2011 at 10:51

**Comment by GoogleCodeExporter** [comment]
Original comment by `boss...@yahoo.fr` on 23 Jul 2011 at 5:00
- Changed state: **WontFix**
